operations/patch_util.go

```diff
@@ -628,10 +628,13 @@ func loadGitData(dir, branch, ref, commits string, format bool, extraArgs ...str
 	// In the case a range is passed, we only need one commit to determine the base, so we use the first commit.
 	// For details see: https://git-scm.com/docs/gitrevisions
 
-	mergeBase, err := gitMergeBase(dir, branch+"@{upstream}", ref, commits)
+	mergeBase, err := gitMergeBase(dir, "upstream/"+branch, ref, commits)
```

Comment:
Can you please add some words to the comment above (on line 635) explaining `upstream/"+branch` in addition to `branch@{upstream}`?

Reply:
Yes! let me add some wording